packages/chrome/src/styled/sheet/StyledSheet.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet';
+
+interface IStyledSheetProps {
+  placement: 'start' | 'end';
+}
+
+const sheetSmartBorderStyle = ({
+  theme,
+  placement
+}: IStyledSheetProps & ThemeProps<DefaultTheme>) => {
+  let borderSide = '';
+
+  if (placement === 'end') borderSide = '-left';
+  else if (placement === 'start') borderSide = '-right';
+
+  return `border${borderSide}: 1px solid ${theme.palette.grey[300]};`;
+};
+
+export const StyledSheet = styled.aside.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  tabIndex: -1
+})<IStyledSheetProps>`
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  order: 1;
+
+  transition: width 0.25s ease-in-out;
```

Comment:
I believe [`transform`](https://csstriggers.com/transform) can be used in this situation instead of [`width`](https://csstriggers.com/width).  The `transform` property does not trigger geometry changes or painting and can be carried out by the compositor thread with the help of the GPU.

Reply:
I didn't know there was a difference between `transform` and `transition`. Thanks for pointing that out, we'll definitely look into it and change it.

Reply:
This makes sense to use `transform` instead of `width`, will look into this!

Reply:
@Francois-Esquire @exelarios 

On second thought, I think transform doesn't move other elements adjacent to it. I think `width` is probably correct here. I'm going to resolve this comment. I think the focus is to address [this comment](https://github.com/zendeskgarden/react-components/pull/1231#discussion_r744257990) instead.